docker-cloud.yml

```diff
@@ -89,4 +89,19 @@ specstore:
   expose:
     - '8000'
   image: 'datopian/specstore:latest'
-
+elasticsearch:
+  autoredeploy: true
+  image: 'elasticsearch:latest'
+  restart: always
+metastore:
+  autoredeploy: true
+  restart: always
+  links:
+    - elasticsearch
+  environment:
+    VIRTUAL_HOST: ${DOMAIN_API}/metastore/*
+    DATAHUB_ELASTICSEARCH_ADDRESS: elasticsearch:9200
+    PRIVATE_KEY:
```

Comment:
Not sure why, but os-condactor search uses PRIVATE_KEY
https://github.com/openspending/os-conductor/blob/master/conductor/blueprints/search/blueprint.py#L5